packages/build/webpack/app.view.template.xml

```diff
@@ -23,6 +23,7 @@
     <% } %>
     <dependencies>
         <dependency path="clientapi"/>
+        <dependency path="passwordGauge.js"/>
```

Comment:
Since we're importing a `.js` file directly here I wonder if it is getting minified in the build? I know that client libraries get minified but am unsure about direct declarations.